src/components/Logo/Logo.tsx

```diff
@@ -35,12 +42,27 @@ export interface Props {
  *
  * Branding image or text of the site.
  */
-export const Logo = ({ alt, className, href, src, title, ...other }: Props) => {
+export const Logo = ({
+  alt,
+  className,
+  children,
+  href,
+  src,
+  ariaLabel,
+  title,
+  ...other
+}: Props) => {
   const componentClassName = clsx(styles['logo'], className, {});
   return (
     <div className={componentClassName} {...other}>
-      <a className={styles['logo__link']} href={href} rel="home" title={title}>
-        <LogoImage alt={alt} src={src} />
+      <a
+        aria-label={ariaLabel}
+        className={styles['logo__link']}
+        href={href}
+        rel="home"
+        title={title}
```

Comment:
Do we need both an `ariaLabel` and a `title`?

Reply:
title probably wouldn't hurt so a user could hover and get a message there. I don't know what other value that would add